lua/nvim_lsp/metals.lua

```diff
@@ -0,0 +1,79 @@
+local skeleton = require 'nvim_lsp/skeleton'
+local util = require 'nvim_lsp/util'
+local lsp = vim.lsp
+local server_name = "metals"
+local bin_name = "metals"
+
+local function make_installer()
+  local install_dir = util.path.join{util.base_install_dir, server_name}
+  local metals_bin = util.path.join{install_dir, bin_name}
+  local X = {}
+  function X.install()
+    local install_info = X.info()
+    if install_info.is_installed then
+      print(name, "is already installed")
```

Comment:
Out of curiosity, how to updates work? If a user has an old version installed, this will just tell them an old version is installed? Would they have to remove it, and then issue another install to get the new version installed?

Reply:
> Would they have to remove it, and then issue another install to get the new version installed?

yes